### PR TITLE
Core haptics generator

### PR DESCRIPTION
**Pull request by OoreFasawe**
## Description
- Implemented core haptics to make a quick vibration pattern anytime the user changes their attendance for a party. Vibration resembles the vibration made when liking a tweet on Twitter Mobile for reference.
- In further PRs, we will handle cases where the pattern is not able to play due to possible failures described in the pull request source code

## Milestones
- Added technically challenging feature not discussed during the training sessions.

## Test Plan
- Video demo unavailable but one can run the app on their device to see the change. 5 of my peers tested the feature with the company Iphone; 100% positive feedback received. Overarching comment was that it gave the app a "cleaner" feel.

### CODE REVIEW

File: ThrowIT/ViewControllers/TimelineViewController.m [hunk 1]
```diff
@@ -91,14 +92,14 @@ -(void)fetchParties{
...
-                    }}];
-            }
+//            if(self.distanceDetailsList.count != self.partyList.count){
+//                self.distanceDetailsList = [Utility getDistancesFromArray:self.partyList withCompletionHandler:^(BOOL success) {
+//                    if(success){
+//                        dispatch_async(dispatch_get_main_queue(), ^{
+//                            [self.tableView reloadData];
+//                        });
+//                    }}];
+//            }
```
Comment (jcarlos1202) [inline ThrowIT/ViewControllers/TimelineViewController.m]: Remove commented out code

File: ThrowIT/Haptics/CoreHapticsGenerator.m [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+@implementation CoreHapticsGenerator
+
++(instancetype)initWithEngineOnViewController:(UIViewController *)viewController{
+    if(CHHapticEngine.capabilitiesForHardware.supportsHaptics){
+        NSError* error;
+        CoreHapticsGenerator *myGenerator = [[CoreHapticsGenerator alloc] init];
+        myGenerator.engine = [[CHHapticEngine alloc] initAndReturnError:&error];
+        __weak CoreHapticsGenerator *weakMyGenerator = myGenerator;
+        [myGenerator.engine setResetHandler:^{
+            NSLog(@"Engine RESET!");
```
Comment (jcarlos1202) [inline ThrowIT/Haptics/CoreHapticsGenerator.m]: Remove debugging logs

**Review by jcarlos1202 — CHANGES_REQUESTED** [review]
Added some comments

**Review by OoreFasawe — COMMENTED** [review]
Resolved changes

**Review by jcarlos1202 — APPROVED** [review]
LGTM!

**Pull request merged**